monitoring/sql/estimated-current.sql

```diff
@@ -0,0 +1,16 @@
+CREATE TABLE IF NOT EXISTS `monitoring_billing_mark` (
+  `mark` BIGINT
+) ENGINE = InnoDB;
+
+INSERT INTO `monitoring_billing_mark` (mark) VALUES (NULL);
+
+CREATE TABLE IF NOT EXISTS `monitoring_billing_data` (
+  `year` INT NOT NULL,
+  `month` INT NOT NULL,
+  `service_description` VARCHAR(100) NOT NULL,
+  `sku_description` VARCHAR(100) NOT NULL,
```

Comment:
I'm a bit uncomfortable that we're using "descriptions" rather than some, like, unique name. Can you comment on why descriptions are OK? One concrete concern I have is, what if google changes the description name, that will make it harder to link costs from before and after the description change.

Reply:
This is the schema of the google billing table. I could add the sku.id and service.id to the tables we store, but I think we want to display the descriptions.   https://cloud.google.com/billing/docs/how-to/export-data-bigquery-tables#data-schema